src/Components/Pages/Home/Reviews/ReviewCard.module.scss

```diff
@@ -10,14 +16,61 @@
   transition-duration: 0.4s;
 }
 
+.review h1,
+p {
+  font-family: "Montserrat", sans-serif;
+  font-weight: 400;
+  font-size: 1rem;
+}
+
+.review h1 {
+  font-weight: 600;
+}
+
+.quote {
+  opacity: 0.6;
+  transition-duration: 0.4s;
+}
+
+.review:hover .quote {
+  opacity: 0.6;
+  filter: invert(100%);
+  transition-duration: 0.4s;
+}
+
+@media screen and (min-width: 300px) {
+  .review {
+    height: 100vh;
+  }
+}
+
+@media screen and (min-width: 370px) {
+  .review {
+    height: 80vh;
+  }
+}
+
+@media screen and (min-width: 400px) {
+  .review {
+    height: 70vh;
+  }
+}
+
+@media screen and (min-width: 500px) {
+  .review {
+    height: 65vh;
+  }
+}
+
 @media screen and (min-width: 760px) {
   .review {
-    width: 60vw;
+    width: 65vw;
   }
 }
 
-@media screen and (min-width: 1000px) {
+@media screen and (min-width: 1000px) and (min-height: 500px) {
   .review {
-    width: 20vw;
+    width: 40vw;
+    height: 75vh;
```

Comment:
![image](https://github.com/nitsmun/nitsmun2023-24/assets/106585924/e23de34b-ec95-4dda-b6cb-656a7a14cb01)
30 or35 vh height is sufficient.